testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergRedirectionToHive.java

```diff
@@ -357,16 +357,14 @@ public void testCommentTable()
         createHiveTable(hiveTableName, false);
 
         assertTableComment("hive", "default", tableName).isNull();
-        // TODO: support redirects from Iceberg to Hive
-        assertThat(readTableComment("iceberg", "default", tableName)).hasNoRows();
```

Comment:
Looks like this fixes a (small) bug, right?

Reply:
Without redirections hiding the non-Iceberg tables could have been nice to have, at the cost of making `drop schema` sometimes fail in confusing ways

Once we have redirections, hiding these tables would certainly be a bug